spark/src/test/java/org/apache/zeppelin/spark/SparkInterpreterTest.java

```diff
@@ -92,14 +96,40 @@ public void setUp() throws Exception {
       repl.open();
     }
 
+    final RemoteEventClientWrapper remoteEventClientWrapper = new RemoteEventClientWrapper() {
+
+      @Override
+      public void onParaInfosReceived(String noteId, String paragraphId,
+          Map<String, String> infos) {
+        if (infos != null) {
+          paraIdToInfosMap.put(paragraphId, infos);
+        }
+      }
+
+      @Override
+      public void onMetaInfosReceived(Map<String, String> infos) {
+      }
+    };
     context = new InterpreterContext("note", "id", null, "title", "text",
         new AuthenticationInfo(),
         new HashMap<String, Object>(),
         new GUI(),
         new AngularObjectRegistry(intpGroup.getId(), null),
         new LocalResourcePool("id"),
         new LinkedList<InterpreterContextRunner>(),
-        new InterpreterOutput(null));
+        new InterpreterOutput(null)) {
+        
+        @Override
+        public RemoteEventClientWrapper getClient() {
```

Comment:
#1637 also exposed one of method of RemoteEventClient to interpreter. But little different way.
#1637 created new class `RemoteWorksController` to wrap RemoveEventClient.
https://github.com/apache/zeppelin/pull/1637/files#diff-5f11952b2fbf4e41e273df20c409ec4b

This approach and #1637 's approach is basically the same, but in the future refactoring, i think we need to decide either a) create new class for every feature exported from RemoveEventClient (like RemoteWorksController), or b) create a common wrapper class RemoteEventClientWrapper and export all necessary methods from it.